components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/java/org/wso2/carbon/identity/api/server/userstore/v1/core/ServerUserStoreService.java

```diff
@@ -238,35 +238,24 @@ public UserStoreConfigurationsRes getUserStoreByDomainId(String domainId) {
     }
 
     /**
-     * Retrieve the meta of configured user store type.
+     * Retrieve the meta of user store type.
      *
      * @param typeId the user store type id.
-     * @param limit  items per page.
-     * @param offset to specify the offset param.
-     * @param filter to specify the filtering capabilities.
-     * @param sort   to specify the sorting order.
      * @return MetaUserStoreType.
      */
 
-    public List<MetaUserStoreType> getUserStoreManagerProperties(String typeId, Integer limit, Integer offset,
-                                                                 String filter, String sort) {
-
-        handleNotImplementedBehaviour(limit, offset, filter, sort);
+    public MetaUserStoreType getUserStoreManagerProperties(String typeId) {
```

Comment:
why were the pagination + sorting params removed?

Reply:
Previously pagination and sorting have been added because it retrieves an array containing all configured userstores which are in the defined userstore type. Thus, the same set of properties which can be configured for that userstore type is repeated.
ex: If we use "typeId": "VW5pcXVlSURKREJDVXNlclN0b3JlTWFuYWdlcg" which maps with
 "typeName": "UniqueIDJDBCUserStoreManager", and there are 20 userstores configured in that type, the response contains all of these userstores properties. We can use pagination to take 10 results per page. Similarly, sorting can be used to sort the response according to user store name, etc. However, this is not supported yet.

Since now we provide only the set of properties for the particular user store type, pagination and sorting are not required.